src/renderer/features/network/CustomRpcForm/model/__test__/add-custom-rpc-model.test.ts

```diff
@@ -0,0 +1,106 @@
+import { fork, allSettled } from 'effector';
+
+import { addCustomRpcModel } from '../add-custom-rpc-model';
+import { ConnectionStatus } from '@shared/core';
+import { ExtendedChain } from '@entities/network';
+import { RpcCheckResult } from '../../lib/types';
+
+describe('features/network/CustomRpcForm/add-custom-rpc-model', () => {
+  test('should have error for wrong node url', async () => {
+    const scope = fork({});
```

Comment:
This is also a valid expression 😉
```suggestion
    const scope = fork();
```

Reply:
Thanks, ran a replace it across the whole code base :)